ci/provisioning/post_provision_config_nodes_EL_7.sh

```diff
@@ -95,8 +95,13 @@ EOF
     fi
 
     # CORCI-1096
+    grep 'relayhost' /etc/postfix/main.cf
     echo 'relayhost = [mail.wolf.hpdd.intel.com]' >> /etc/postfix/main.cf
-    postfix reload
+    if ! postfix reload; then
+        postfix reload
```

Comment:
Seems to be an extraneous `postfix reload` left in here.  It's not really causing any harm other than possibly masking the error code of the `postfix reload` that actually failed.  But ultimately we still have better visibility that `postfix reload` is actually failing and the main portion of this patch is the much more important goal of the PR.  I will remove this extraneous `postfix reload` in a future PR.

Reply:
Seems like we should just remove it now.  Since we've seen a few `postfix_reload` problems lately wouldn't it be good to get the actual error code from the failing command?